### ISSUE TITLE
feature: add IsStrongPassword decorator

### ISSUE THREAD

**Issue by jhullse** [typestack/class-validator#1024]
### Description
Validating strength of passwords is a very common requirement for applications. At this moment `class-validator` doesn't support that even though `validator.js` already does.

### Proposed solution
Add the `IsStrongPassword` decorator to wrap the functionality already provided by `validator.js`.

**Comment by braaar** [comment]
Why was this reopened, @NoNameProvided? Waiting for release?

**Comment by NoNameProvided** [comment]
Yes, the fixed label is added to issues merged into develop, but not released. I write the changelog via filtering for fixed labels and close the issues only after the release.

**Comment by NoNameProvided** [comment]
This feature/fix has been released in `0.14.0`.
